src/components/InsureeFilter.js

```diff
@@ -29,6 +29,10 @@ const styles = (theme) => ({
 const INSUREE_FILTER_CONTRIBUTION_KEY = "insuree.Filter";
 
 class InsureeFilter extends Component {
+    constructor(props) {
+        super(props);
+        this.isWorker = props.modulesManager.getConf("fe-insuree", "isWorker", false);
```

Comment:
Consider moving the default value to the constants. By doing so, you can ensure a single source of truth, especially since this variable is used across multiple components and its default value might vary. The best approach would be to create a DEFAULT object, which will contain this variable:
`
const DEFAULT = {
IS_WORKER: false,
};
`
And then you can use it like this:
`
this.isWorker = props.modulesManager.getConf("fe-insuree", "isWorker", DEFAULT.IS_WORKER);
`

Reply:
good point

Reply:
done